config/lib/helpers/topic.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
+const contentTypes = {
```

Comment:
I think it would be a good idea to add some comments to explain why we are exposing broadcasts here as topics?